ksql-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -38,24 +49,31 @@
   private final Endpoints endpoints;
   private final HttpServerOptions httpServerOptions;
   private final Server server;
+  private final boolean proxyEnabled;
   private ConnectionQueryManager connectionQueryManager;
-  private HttpServer httpServer;
+  private volatile HttpServer httpServer;
+
+  private HttpClient proxyClient;
+  private SocketAddress proxyTarget;
 
   public ServerVerticle(final Endpoints endpoints, final HttpServerOptions httpServerOptions,
-      final Server server) {
+      final Server server, final boolean proxyEnabled) {
     this.endpoints = Objects.requireNonNull(endpoints);
     this.httpServerOptions = Objects.requireNonNull(httpServerOptions);
     this.server = Objects.requireNonNull(server);
+    this.proxyEnabled = proxyEnabled;
   }
 
   @Override
   public void start(final Promise<Void> startPromise) {
+    this.proxyClient = proxyEnabled ? vertx
+        .createHttpClient(
+            new HttpClientOptions().setMaxPoolSize(10).setMaxInitialLineLength(65536)) : null;
```

Comment:
What's the purpose of `setMaxInitialLineLength()` here?

Reply:
It shouldn't be needed. Will remove.